net/nginx/files-luci-support/luci_nginx.conf

```diff
@@ -1,5 +1,5 @@
 
-user  root;
+user nobody nogroup;
```

Comment:
separate commit please

Reply:
didn't want to create 2 commit for that small change but if it's a problem i will split them then...

Reply:
It's okay to have multiple commits in single pull request